src/Server/Http/Controllers/OAuthRegisterController.php

```diff
@@ -37,7 +37,7 @@ public function __invoke(Request $request): JsonResponse
         );
 
         $client = $clients->createAuthorizationCodeGrantClient(
-            name: $request->get('name'),
+            name: $request->get('client_name', $request->get('name')),
```

Comment:
imho we can remove the fallback, because it's not according to the spec. But we can check what Taylor prefers

Reply:
@taylorotwell @joetannenbaum what do you think about this?
I’m leaning toward removing the fallback, but that would be a breaking change unless we’re okay introducing it now or want to handle it with a fallback in the interim.